File: Book.py
class Book:
    def __init__(self, name: str, author: str, isbn: int, rentaling: bool, location: str):
        __self.name: str = name
        __self.author: str = author
        __self.isbn: int = isbn
        __self.rentaling: bool = rentaling
        __self.location: str = location

    def getName(self) -> str:
        return __self.name

    def getAuthor(self) -> str:
        return __self.author

    def getIsbn(self) -> int:
        return __self.isbn

    def getRentaling(self) -> bool:
        return __self.rentaling

    def getLocation(self) -> str:
        return __self.location

File: library-server/src/main/kotlin/com/example/libraryserver/domain/book/Book.kt
package com.example.libraryserver.domain.book

import com.example.libraryserver.domain.AuditEntity
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Table


@Entity
@Table(name = "Book")
class Book(
    name: String,
    author: String,
    isbn: String,
    isRental: Boolean,
    location: String
): AuditEntity() {
    @Column(name = "name")
    var name: String = name

    @Column(name = "author")
    var author: String = author

    @Column(name = "isbn")
    var isbn: String = isbn

    @Column(name = "is_rental")
    var isRental: Boolean = isRental

    @Column(name = "location")
    var location: String = location

    override fun toString(): String =
        "${this.name}, ${this.author}, ${this.isbn}, ${this.isRental} ${this.location}"

    fun toDto(): BookDto =
        BookDto(
            name = this.name,
            author = this.author,
            isbn = this.isbn,
            isRental = this.isRental,
            location = this.location
        )

    fun toResDto(): BookResDto =
        BookResDto(
            id = this.id!!,
            name = this.name,
            author = this.author,
            isbn = this.isbn,
            isRental = this.isRental,
            location = this.location
        )
}